lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -2387,6 +2394,11 @@ std::istream &SPIRVModuleImpl::parseSPIRV(std::istream &I) {
       SPIRVDBG(spvdbgs() << "getWordCountAndOpCode EOF 0 0\n");
       break;
     }
+    if constexpr (IS_REPORT) {
+      if (OpCode == OpMemoryModel) {
```

Comment:
That does look fishy, why do we need to skip OpMemoryModel? There should be no more then 2 OpMemoryModel in the model.